.github/workflows/ci-components.yml

```diff
@@ -84,6 +84,8 @@ jobs:
           cache-dependency-path: yarn.lock
       - name: Install Dependencies
         run: yarn install --frozen-lockfile
+      - name: WTH is happening
+        run: tree node_modules/@hashicorp/design-system-tokens
```

Comment:
/dist is there

Now I do get the same CI errors on `yarn run test:ember-compatibility`